addons/rose/package.json

```diff
@@ -30,7 +30,7 @@
   },
   "dependencies": {
     "@embroider/test-setup": "^2.1.1",
-    "@hashicorp/design-system-components": "^3.5.0",
+    "@hashicorp/design-system-components": "^4.0.0",
```

Comment:
quick question: currently this package is at `4.4.1`, any specific reason why to upgrade to `4.0.0` instead? (all the updates from `4.0.0` to `4.4.1` are minors, so there should be no difference in the outcome, and you would have the latest available version)

Reply:
I guess it might have been better to be explicit but in practice it didn't matter too much since this pulled in `4.4.1` anyways since this pulled in the latest version.

Reply:
great question, i left it as 4.0.0 because it resolves to 4.4.1 either way. However, i do think it would be better to be more specific. I'll update this.